lesson_06/script.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const fragment = document.createDocumentFragment();
```

Comment:
IMO
I thought it would be better to declare it in Promis that uses the variable fragment to narrow the scope of the variable.